lib/test.js

```diff
@@ -2124,6 +2125,138 @@ queue.push(function () {
   });
 });
 
+// Voice webhook create
+queue.push(function () {
+  var params = {
+    url: 'https://example.com/webhook',
+    token: 'secret-token'
+  };
+
+  nock('https://voice.messagebird.com')
```

Comment:
instead of typing in the host yourself you can make use of the `VOICE_ENDPOINT` variable (I'm guessing by the name it is supposed to be treated as a const)